docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+
+  pipe-rt-manager:
+    container_name: pipe-rt-manager
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
```

Comment:
What is it for?

Reply:
easy to run locally 
docker-compose up -d  --build
instead of 
docker build .....
docker run ......

Reply:
what is the args for?